NeuroID/TrackerClassExtensions/Utils.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Utils.swift
+//  NeuroID
+//
+//  Created by Kevin Sites on 3/30/23.
+//
+
+import Foundation
+import UIKit
+
+internal func neuroIDPasteUtil(caller: UIResponder, view: UIView, text: String?, className: String?) {
+//    neuroIDPasteUtil(caller: caller, view: view, text: text, className: className)
```

Comment:
This used to be called in the swizzle, looked like it would be an endless recursive but I guess not? However when I moved it to its own file then it did warn about recursive calling